packages/web-react/src/components/Card/CardLink.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import classNames from 'classnames';
+import React from 'react';
+import { useClassNamePrefix } from '../../hooks';
+import { SpiritLinkProps } from '../../types';
+import { Link } from '../Link';
+
+const CardLink = (props: SpiritLinkProps) => {
+  const { children, UNSAFE_className, ...restProps } = props;
+  const cardClass = useClassNamePrefix('CardLink');
+
+  return (
+    <Link {...restProps} UNSAFE_className={classNames(cardClass, UNSAFE_className)}>
```

Comment:
```suggestion
    <Link href={href} {...restProps} UNSAFE_className={classNames(cardClass, UNSAFE_className)}>
```

Reply:
? maybe. I dont know if it will work after @curdaj PR is merged

Reply:
@adamkudrna this should be somehow consistent between web and react, please decide how

Reply:
In the current state of things, I think this is OK.